charts/pulsar/templates/autorecovery-statefulset.yaml

```diff
@@ -139,7 +139,7 @@ spec:
         - >
           bin/apply-config-from-env.py conf/bookkeeper.conf;
           {{- include "pulsar.autorecovery.zookeeper.tls.settings" . | nindent 10 }}
-          exec bin/bookkeeper autorecovery
+          OPTS="${OPTS} -Dlog4j2.formatMsgNoLookups=true" exec bin/bookkeeper autorecovery
```

Comment:
I was going to suggest that we update an argument elsewhere in a configmap, but by changing it this way, we guarantee that all users get this JVM arg.